internal/auth0/client/expand.go

```diff
@@ -41,7 +41,7 @@ func expandClient(data *schema.ResourceData) (*management.Client, error) {
 		InitiateLoginURI:                   value.String(config.GetAttr("initiate_login_uri")),
 		EncryptionKey:                      value.MapOfStrings(config.GetAttr("encryption_key")),
 		IsTokenEndpointIPHeaderTrusted:     value.Bool(config.GetAttr("is_token_endpoint_ip_header_trusted")),
-		OIDCBackchannelLogout:              expandOIDCBackchannelLogout(data),
```

Comment:
Instead of removing deprecate it. it's not going to be a major release.

Reply:
The concern I had with that approach is you introduce the potential of setting the back channel logout urls in two different places, via the depreciated property and via the new nested properties. I imagine the Auth0 API will reject if you try that but I wasn't sure?

I also was trying to follow the proposal in this issue https://github.com/auth0/terraform-provider-auth0/issues/1030
Generally I do prefer flat json to nested properties. 

It seems the main reason for this change is your comment about backwards compatibility. These properties exist on the API and are set even though you can't control them via terraform currently. I guess the concern is someone gets a provider update and then it will try to set them back to the default values if they don't add their current values to their terraform? I can see that would be an issue. I just hate resorting to nested JSON but I suppose there isn't a better option.